en/development/errors.rst

```diff
@@ -204,17 +205,19 @@ For more details on HTTP 4xx error status codes see :rfc:`2616#section-10.4`.
 
 For more details on HTTP 5xx error status codes see :rfc:`2616#section-10.5`.
 
+
 You can throw these exceptions from you controllers to indicate failure states,
 or HTTP errors. An example use of the HTTP exceptions could be rendering 404
 pages for items that have not been found::
 
-    public function view($id)
+    public function view($id = null)
     {
-        $post = $this->Post->findById($id);
-        if (!$post) {
-            throw new NotFoundException('Could not find that post');
+        $post = $this->Posts->findById($id)->first();
+        if (empty($post)) {
+            throw new \Cake\Network\Exception\NotFoundException(__('Post not found'));
```

Comment:
This is so that you can copy and paste the snipplet. Should I put a use ontop instead?